SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -446,6 +446,12 @@ void USpatialNetDriver::CreateAndInitializeCoreClasses()
 		MakeUnique<SpatialGDK::ActorSystem>(ActorNonAuthSubview, TombstoneActorSubview, this, &TimerManager, Connection->GetEventTracer());
 	ClientConnectionManager = MakeUnique<SpatialGDK::ClientConnectionManager>(SystemEntitySubview, this);
 
+	const SpatialGDK::FSubView& DebuggerSubView = Connection->GetCoordinator().CreateSubView(
```

Comment:
Rather than doing this always, I think it'd be nicer if this was only done on demand when the debugger is spawned. The spawning flow is a little different, servers spawn it via the gsm, and clients have it replicated over the wire. But seems like a nice use of a subview that it's only setup when it's actually needed.

Reply:
I did initially create it on `ASpatialDebugger::BeginPlay` but moved it here as per @jacquese's suggestion. `Advance` was called on `ASpatialDebugger::Tick` so the subview and the debugger system themselves were contained there; could move it to the NetDriver but this would spread the code across multiple moving parts which could be unnecessarily confusing.